corehq/apps/locations/models.py

```diff
@@ -113,6 +113,27 @@ def can_have_children(self):
         return LocationType.objects.filter(parent_type=self).exists()
 
 
+class LocationQueriesMixin(object):
```

Comment:
By using this mixin, methods we define here can be used either from `SQLLocation.objects`, or after chaining queryset methods together.